Tests/scripts/update_conf_json.py

```diff
@@ -99,19 +149,22 @@ def run():
                         is_yml_file = integration_file.endswith('.yml')
                         file_path = os.path.join(inner_dir_path, integration_file)
                         if is_yml_file:
+                            integration_test_playbooks.extend(get_integration_tests(file_path))
                             pack_integrations.append(get_integration_data(file_path))
                 else:
                     is_yml_file = file_or_dir.endswith('.yml')
                     file_path = os.path.join(integration_dir_path, file_or_dir)
                     if is_yml_file:
                         pack_integrations.append(get_integration_data(file_path))
+                        integration_test_playbooks.extend(get_integration_tests(file_path))
 
         for file_path in os.listdir(test_playbook_dir_path):
             is_yml_file = file_path.endswith('.yml')
             file_path = os.path.join(test_playbook_dir_path, file_path)
             if is_yml_file and find_type(file_path) == FileType.TEST_PLAYBOOK:
                 test_playbook_id, fromversion = get_playbook_data(file_path)
-                if test_playbook_id not in existing_test_playbooks:
+                is_test_relevant = test_playbook_id in set(integration_test_playbooks)
+                if test_playbook_id not in existing_test_playbooks and is_test_relevant:
```

Comment:
I'm not sure if this is robust enough. take the `Panorama` pack for example.
in the .yml only `tests: - palo_alto_firewall_test_pb - palo_alto_panorama_test_pb` are configured.
but I do wish the extra 10 in the TPB related dir to run.

Maybe opt to check if the subplaybook of the PB is also in the pack - but that's a different effort.